content/neuvector-prime/azure/contents.lr

```diff
@@ -0,0 +1,476 @@
+neuvector-scanner-pod-9489c5d78-v5xrq      1/1     Running     0          10s
+neuvector-updater-pod-28294240-lpshs       0/1     Completed   0          10m
+neuvector-updater-pod-28294245-xrk4n       0/1     Completed   0          5m11s
+neuvector-updater-pod-28294250-hqpcg       0/1     Completed   0          11s
+```
+
+
+## Log into the NeuVector console
+
+The manager service type was set to Load Balancer during install. An external hostname has been assigned for accessing the NeuVector console. By default this URL is accessible from the internet. However, your organization may have placed additional restrictions on external access to your cluster. To retrieve details about the load balancer execute the following command in Cloud Shell.
+
+```
+kubectl get svc -n neuvector neuvector-service-webui
+```
+
+The full dashboard url can be retrieved with the following commands.
+
+```
+SERVICE_IP=$(kubectl get svc --namespace neuvector neuvector-service-webui -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+echo https://$SERVICE_IP:8443
+```
+
+This will print out a URL that looks like 
+
+```
+https://10.20.10.1:8443
+```
+
+This URL provides access to the NeuVector console which is running by default on port 8443. Open a browser window, paste the URL.
+The NeuVector login will appear, login with the credential which you have set during offer creation for admin user.
+
+## Navigating the console
+
+Once logged in, you can begin to [navigate and configure NeuVector]. Refer (https://open-docs.neuvector.com/navigation/navigation).
+
+
+**IMPORTANT!**
+
+    ---
+    The current NeuVector deployment exposes the original password in the Cluster
+    configuration settings in Azure Portal. Until this is resolved, it is suggested
+    to change the Admin password after initial login.  Edit profile in the NeuVector
+    console to change password.
+
+    ---
+
+
+
+1. Access NeuVector Console as a admin user
+2. Click Admin user dropdown and click My Profile
+3. Click Edit Profile and set New password and Update Profile.
+
+
+
+
+
+
+   !["Change Admin Password"](./changepwd.png)
+
+
+
+
+
+
+
+
+**IMPORTANT!**
+
+    ---
+    Do not change or save the cluster configuration settings for Fullname, Password,
+    or Role by changing the key value pairs :
+
+           "controller.secret.data.userinitcfg\\.yaml.users[0].Fullname"
+           "controller.secret.data.userinitcfg\\.yaml.users[0].Password"
+           "controller.secret.data.userinitcfg\\.yaml.users[0].Role"
+
+    Changing this value does not update the NeuVector Admin Fullname, Password,
+    or Role. This will cause unnecessary cluster restarts.
+
+    ---
+
+
+
+
+
+
+
+   !["Keyvaluepair config"](./keyvaluepairconfig.png)
+
+## NeuVector Usage Record and Billing
+
+This command will print out CSP Adapter Usage Record:
+
+```
+kubectl get CspAdapterUsageRecord  neuvector-usage -n neuvector -oyaml
+```
+
+it should list similar output
```

Comment:
The adapter usage record output should look similar to the following example:

Reply:
Removed this section